scripts/exglobal_stage_ic.sh

```diff
@@ -59,18 +68,35 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
     # Stage the FV3 cold-start initial conditions to ROTDIR
     YMD=${PDY} HH=${cyc} declare_from_tmpl COM_ATMOS_INPUT
     [[ ! -d "${COM_ATMOS_INPUT}" ]] && mkdir -p "${COM_ATMOS_INPUT}"
+
```

Comment:
Why are there changes in this script?  How are they related to porting the workflow to AWS?